### ISSUE TITLE
webpack config question - no-unresolved false positives

### ISSUE THREAD

**Issue by dmt0** [benmosher/eslint-plugin-import#284]
My problem is that all imports report no-unresolved

I have the dir structure as follows:

```
/web/
  /src
  /webpack/common.config.js
  .eslintrc
  package.json
```

.eslintrc contains this:

    "settings": {
      "import/resolve": { "webpack": { config:webpack/common.config.js" } },
      "import/parser": "babel-eslint",
    },

eslint is exectured from /web as follows:
   
    eslint src

The documentation does not mention anything regarding the webpack configuration (unless I missed something) Is there anything that is required to make this work? My config currently look like this:

    const PATHS = {
      app: path.join(__dirname, '../src'),
      build: path.join(__dirname, '../dist'),
    };

    process.env.BABEL_ENV = TARGET;

    const common = {
      entry: [
        PATHS.app,
      ],

      output: {
        path: PATHS.build,
        filename: 'bundle.js',
        chunkFilename: '[name]-[hash].js',
      },

      resolve: {
        alias: {
          config: path.join(PATHS.app + '/config', process.env.NODE_ENV || 'development')
        },
        extensions: ['', '.jsx', '.js', '.json', '.scss'],
        modulesDirectories: ['node_modules', PATHS.app],
      },
    ...
    }

Is there something missing?

**Comment by benmosher** [comment]
Setting should be

**Comment by dmt0** [comment]
Thanks, fixed those, but issue did not resolve. Yes, the package.json is in the root dir the config is in webpack/common.config.js relative to root dir.

**Comment by benmosher** [comment]
and I assume `module.exports = common` or some such occurs somewhere later in common.config.js?

**Comment by dmt0** [comment]
And that was the problem, I had this:

```
if (TARGET === 'start' || !TARGET) {
  module.exports = merge(development, common);
}

if (TARGET === 'build' || !TARGET) {
  module.exports = merge(production, common);
}
```

So now I added this and it solved it for running `npm run lint`:

```
if (TARGET === 'lint' || !TARGET) {
  module.exports = merge(production, common);
}
```

Thanks!
